src/Compilers/CSharp/Test/Symbol/Symbols/MissingSpecialMember.cs

```diff
@@ -1369,39 +1369,6 @@ public static void Main()
                 );
         }
 
-        [Fact]
-        public void System_String__ConcatObjectObject()
```

Comment:
We no longer trigger these, as we no longer make use of this method, so I removed the tests.